tabuckner/broskichat
# added a basic contact page

### Project Changed


**- What I Did**
Added a basic contact page

**- How I Did It**
Bulma Tiles are lit but we could play around with colors/spacing later. Later down the road if we used custom css for bulma, I would change the padding and margins of the form.

**- How to Test/Verify It**
Routing should be all good to go so yarn start the API and then the App, just click the contact button on the header!


**- An Image to Make Us Giggle (not mandatory but encouraged)**
